portfolio/src/main/webapp/script.js

```diff
@@ -83,26 +83,26 @@ window.addEventListener('scroll', function() {
     document.getElementById('navbar').style.padding = '30px 10px';
     document.getElementById('navbar').style.backgroundColor = 
         'rgba(0, 0, 0, 0.25)';
-    document.getElementById('logo').style.fontSize = '25px';
+    document.getElementById('myName').style.fontSize = '25px';
   }
   else {
     document.getElementById('navbar').style.padding = '80px 10px';
     document.getElementById('navbar').style.backgroundColor = 
         'rgba(0, 0, 0, 1)';
-    document.getElementById('logo').style.fontSize = '35px';
+    document.getElementById('myName').style.fontSize = '35px';
   }
 })
 
 /**
  * Fetches a message from the server and adds it to the DOM.
```

Comment:
Update comment, seems like you're fetching more messages than one now! You might want to make it clear in this comment too that the message is fetched from the messages submitted via the form.

Reply:
Done!